custom/icds_reports/reports/incentive.py

```diff
@@ -43,7 +43,7 @@ def get_excel_data(self):
                     ['Generated at', india_now()],
                     ['Grouped By', 'AWC'],
                     ['Month', self.month.month],
-                    ['Year', self.month.year]
+                    ['Year', self.month.year],
```

Comment:
:grimacing:

Reply:
yeah, a bunch of little things that were apparently not invalid syntax but did not at all do the right thigns